flask_helpers.py

```diff
@@ -1,33 +1,108 @@
 import os
+import re
 from functools import wraps
 
 from flask import redirect, render_template, session
 
+from logger import LOG
+
 
 def is_admin_interface():
     return os.getenv("ADMIN", "false") == "true"
 
 
+def has_admin_role():
+    is_admin_role = False
+    group = session.get("group", None)
+    if group:
+        is_admin_role = "admin-" in group["value"]
```

Comment:
I suspect you'll know more than I about this kind of thing so feel free to disagree. While I'll concede it's unlikely, it is theoretically possible that in the future someone could add a new group containing "admin-" (say "local-authority-admin-barnet") that isn't meant to have admin control over the entire app (in this example, just a subset of users) and wouldn't necessarily check through the code to see if it has side effects - causing unintended security problems. I suggest this be changed to something like this which is more explicit:
```
is_admin_role = group["value"] in ["admin-view",  "admin-power", "admin-full"]
```

Reply:
Good catch. I'll change.